src/SFA.DAS.Forecasting.AcceptanceTests/Payments/Steps/ProcessPaymentEventSteps.cs

```diff
@@ -148,26 +148,18 @@ public void ThenTheForecastingPaymentServiceShouldStoreThePaymentDeclarationsRec
         {
             WaitForIt(() =>
             {
-                Console.WriteLine($"Looking for Payments. Employer Account Id: {Config.EmployerAccountId}");
-                var parameters = new DynamicParameters();
-                parameters.Add("@employerAccountId", receivningEmployerId, DbType.Int64);
-                parameters.Add("@sendingEmployerAccountId", sendingEmployerId, DbType.Int64);
-                var sql = @"
-                        Select* from Payment 
-                        where employerAccountId = @employerAccountId 
-                        and SendingEmployerAccountId = @sendingEmployerAccountId 
-                        and FundingSource = 2
-                        ";
-                var payments = Connection.Query<PaymentModel>(sql, parameters, commandType: CommandType.Text).ToList();
+                var payments = DataContext.Payments
+                    .Where(m => m.EmployerAccountId == receivningEmployerId
+                             && m.SendingEmployerAccountId == sendingEmployerId
+                             && m.FundingSource == FundingSource.Transfer);
 
-                foreach (var payment in Payments)
-                {
-                    if (payments.Any(p => payment.PaymentId == p.ExternalPaymentId)) continue;
-                    Console.WriteLine($"Payment not found. Payment: {payment.ToJson()}");
-                    return false;
-                }
-                return true;
-            }, "Failed to find all the payments.");
+                var paymentsSaved = Payments.Count(p => payments.Any(expected => expected.ExternalPaymentId == p.PaymentId));
+                // ToDo: Move to failed text
+                Console.WriteLine($"{paymentsSaved} of expected {Payments.Count()} Payment found.");
+
+                return Payments.All(p => payments.Any(expected => expected.ExternalPaymentId == p.PaymentId));
```

Comment:
This code no longer outputs the payments that have failed to save.

Reply:
@abroodie Where should this output?